server/src/test/java/com/objectcomputing/checkins/services/fixture/RepositoryFixture.java

```diff
@@ -16,6 +16,7 @@
 import com.objectcomputing.checkins.services.guild.member.GuildMemberRepository;
 import com.objectcomputing.checkins.services.member_skill.MemberSkillRepository;
 import com.objectcomputing.checkins.services.memberprofile.MemberProfileRepository;
+import com.objectcomputing.checkins.services.onboardeeprofile.OnboardingProfileRepository;
```

Comment:
We will need a similar (with less in it) file for onboard-server, and this onboarding profile registry line will not be needed in the main server (unless I am mistaken, and they are needed in 2 different apps, but I don't think so.)

It is worth considering however what manner we will use to access the data on onboarding profiles from the database, whether the database should even be shared, etc, between the apps.

Reply:
Repository Fixture was already present.